src/migration-scripts/ipoe-server/1-to-2

```diff
@@ -39,12 +44,39 @@ with open(file_name, 'r') as f:
 
 config = ConfigTree(config_file)
 base = ['service', 'ipoe-server']
-pool_base = base + ['client-ip-pool']
+
 if not config.exists(base):
     exit(0)
 
+if config.exists(base + ['authentication', 'interface']):
+    for interface in config.list_nodes(base + ['authentication', 'interface']):
+        config.rename(base + ['authentication', 'interface', interface, 'mac-address'], 'mac')
+
+        mac_base = base + ['authentication', 'interface', interface, 'mac']
+        for mac in config.list_nodes(mac_base):
+            vlan_config = mac_base + [mac, 'vlan-id']
+            if config.exists(vlan_config):
+                config.rename(vlan_config, 'vlan')
+
+for interface in config.list_nodes(base + ['interface']):
+    base_path = base + ['interface', interface]
+    for vlan in ['vlan-id', 'vlan-range']:
+        if config.exists(base_path + [vlan]):
+            print(interface, vlan)
```

Comment:
Remove debugging ? Not that it matters too much.

Reply:
fixed